sources/packages/backend/workflow/src/workflow-definitions/fulltime-assessment-2022-2023.bpmn

```diff
@@ -841,17 +841,18 @@ dmnFullTimeIncomeThreshold</bpmn:text>
       <bpmn:extensionElements>
         <zeebe:ioMapping>
           <zeebe:output source="=if studentDataLivingathomeRent = null&#10;then &#34;no&#34;&#10;else&#10;  studentDataLivingathomeRent" target="studentDataLivingathomeRent" />
-          <zeebe:output source="=if studentDataTransportationCost = null&#10;then 0&#10;else&#10;  studentDataTransportationCost" target="studentDataTransportationCost" />
+          <zeebe:output source="=if studentDataReturnTripHomeCost = null&#10;then 0&#10;else&#10;  studentDataReturnTripHomeCost" target="studentDataReturnTripHomeCost" />
           <zeebe:output source="=if studentDataDaycareCosts11YearsOrUnder = null&#10;then 0&#10;else&#10;  studentDataDaycareCosts11YearsOrUnder" target="studentDataDaycareCosts11YearsOrUnder" />
           <zeebe:output source="=if studentDataDaycareCosts12YearsOrOver = null&#10;then 0&#10;else&#10;  studentDataDaycareCosts12YearsOrOver" target="studentDataDaycareCosts12YearsOrOver" />
           <zeebe:output source="=if studentDataChildSupportAndOrSpousalSupport = null&#10;then 0&#10;else&#10;  studentDataChildSupportAndOrSpousalSupport" target="studentDataChildSupportAndOrSpousalSupport" />
           <zeebe:output source="=if studentDataScholarshipAmount = null&#10;then 0&#10;else&#10;  studentDataScholarshipAmount" target="studentDataScholarshipAmount" />
           <zeebe:output source="=if studentDataGovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataGovernmentFundingCosts" target="studentDataGovernmentFundingCosts" />
-          <zeebe:output source="=if studentDataNongovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataNongovernmentFundingCosts" target="studentDataNongovernmentFundingCosts" />
+          <zeebe:output source="=if studentDataNonGovernmentFundingCosts = null&#10;then 0&#10;else&#10;  studentDataNonGovernmentFundingCosts" target="studentDataNonGovernmentFundingCosts" />
           <zeebe:output source="=if studentDataParentVoluntaryContributionsCosts = null&#10;then 0&#10;else&#10;  studentDataParentVoluntaryContributionsCosts" target="studentDataParentVoluntaryContributionsCosts" />
           <zeebe:output source="=if studentDataIncomeAssistanceAmount = null&#10;then 0&#10;else &#10;  studentDataIncomeAssistanceAmount" target="studentDataIncomeAssistanceAmount" />
           <zeebe:output source="=if studentDataStudentParentNetContribution = null&#10;then 0&#10;else studentDataStudentParentNetContribution" target="studentDataStudentParentNetContribution" />
           <zeebe:output source="=if studentDataVoluntaryContributions = null then 0&#10;else studentDataVoluntaryContributions" target="studentDataVoluntaryContributions" />
+          <zeebe:output source="=if studentDataBCIncomeAssistanceForDisabilitiesCosts = null&#10;then 0&#10;else&#10;  studentDataBCIncomeAssistanceForDisabilitiesCosts" target="studentDataBCIncomeAssistanceForDisabilitiesCosts" />
```

Comment:
This null check code for `studentDataBCIncomeAssistanceForDisabilitiesCosts` is missing on other PYs. Is there a reason?

Reply:
There was some changes to be pushed. That variable is now `studentDataIncomeAssistanceAmount` and we have a null check for that. Thanks!